docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,192 @@
+                value: ""
+              eventType:
+                property: type
+                type: exact
+                value: sap.kyma.custom.demo-app.order.created.v1
+apiRules:
+    - name: function-practical-filip5
+      gateway: kyma-gateway.kyma-system.svc.cluster.local
+      service:
+        host: path.34.90.136.181.xip.io
+        port: 80
+      rules:
+        - methods:
+            - GET
+            - POST
+            - PUT
+            - PATCH
+            - DELETE
+            - HEAD
+          accessStrategies: []
+        - path: /path1/something1
+          methods:
+            - PUT
+            - PATCH
+            - DELETE
+          accessStrategies:
+            - handler: noop
+        - path: /path1/something2
+          methods:
+            - GET
+          accessStrategies:
+            - config:
+                required_scope: ["read"]
+              handler: oauth2_introspection
+        - path: /path2
+          methods:
+            - DELETE
+          accessStrategies:
+              handler: jwt
+            - config:
+                jwksUrls:
+                    - http://dex-service.kyma-system.svc.cluster.local:5556/keys
+                    - http://dex-service.kyma-system.svc.cluster.local:5556
+                trustedIssuers:
+                    - https://dex.34.90.136.181.xip.io
+                    - https://dex.34.90.136.181.xip.io
+env:
+    - name: REDIS_PASS
+      value: YgJUg8z6eA
+    - name: REDIS_PORT
+      value: "6379"
+    - name: REDIS_HOST
+      value: hb-redis-enterp-6541066a-edbc-422f-8bef-fafca0befea8-redis.testme.svc.cluster.local
+    - valueFrom:
+        configMapKeyRef:
+          Name: configmap1
+          Key: token-field
+    - valueFrom:
+        secretKeyRef:
+          Name: secret1
+          Key: token-field
+```
+
+## Specification for a Git Function
+
+See the sample `config.yaml` for a [Git Function](https://kyma-project.io/docs/main/components/serverless/#details-git-source-type) for which code and dependencies are stored in a selected Git repository.
+
+
+```yaml
+name: function-practical-marcin
+namespace: iteration-review
+runtime: nodejs12
+source:
+    sourceType: git
+    url: https://github.com/username/public-gitops.git
+    repository: my-repo
+    reference: master
+    baseDir: /
+    credentialsSecretName: secret2
+```
+
+## Parameters
+
+See all parameter descriptions:
+
+| Parameter         | Required | Related custom resource | Default value  | Description|
+| ---------------------------------------- | :------------: | ---------| ---------| ------- |
+| **name**              | Yes | Function | | Specifies the name of your Function.         |
+| **namespace**             | No | Function | `default` | Defines the Namespace in which the Function is created.        |
+| **runtime**             | Yes | Function | | Specifies the execution environment for your Function. The available values are `nodejs12`, `nodejs14`, and `python38`.      |
```

Comment:
What do you think that add information `nodejs14 is default`?

Reply:
but it does not default 😃  It's generated by default but it's not defaulted to `nodejs14`